entrypoint.sh

```diff
@@ -52,7 +52,8 @@ SCRIPT_AFTER_MAKE_TEST="`pwd`/.github/ci/after_make_test.sh"
 SCRIPT_AFTER_MAKE_TEST_VERSIONED="`pwd`/.github/ci-$SYSTEM_VERSION/after_make_test.sh"
 
 # Infer package name from GITHUB_REPOSITORY
-PACKAGE=$(echo "$GITHUB_REPOSITORY" | sed 's:.*/::' | sed 's:ign-:ignition-:')
+# TODO(jrivero) remove ign renaming after migrate all projects to gazebo-
```

Comment:
The project names should be `gz` just like the repository for simplicity going forward. So I think this change isn't necessary?

Reply:
In that case, It is not, indeed.